tests/unit/app/databaseService.test.ts

```diff
@@ -0,0 +1,229 @@
+import { FieldPacket, RowDataPacket } from 'mysql2/promise';
+import {
+  getTestResultsByVin,
+  getTestResultsByTestId,
+  getTestResultsByVrm,
+  getVehicleDetailsByTrailerId,
+  getVehicleDetailsByVin,
+  getVehicleDetailsByVrm,
+} from '../../../src/app/databaseService';
+import * as technicalQueries from '../../../src/app/queries/technicalRecord';
+import * as testQueries from '../../../src/app/queries/testRecords';
+
```

Comment:
Do we also want to test `getVehicleDetails` it seems to be a dependency of all of our exported api/functions for our databaseService and is responsible for creating the object containing the `result` and `id`. I am thinking that if we were to modify it, it would break those 6 main functions and we wouldn't easily know?
What do you think?

Reply:
It's tested implicitly by calling the `getVehicleDetailsByX` functions, they all proxy to it cause they're basically the same but with different initial calls. So if it was modified, those tests would break anyway.